src/org/parosproxy/paros/network/SSLConnector.java

```diff
@@ -572,6 +573,9 @@ public void decorate(SSLSocket sslSocket) {
 				readSupportedProtocols(sslSocket);
 			}
 			sslSocket.setEnabledProtocols(getClientEnabledProtocols());
+			if (!relaxedTrust) {

```

Comment:
The `relaxedTrust` should not be negated, we want to use all cipher suites when `relaxedTrust` is `true`.